services/base-jdbc/src/main/resources/org/eclipse/hono/service/base/jdbc/store/tenant/base.sql.yaml

```diff
@@ -99,3 +99,11 @@ updateVersioned: |
    AND
       version=:expected_version
 
+getTenants: |
+   SELECT
+      *
+   FROM
+      %s
+   ORDER BY tenant_id
+   LIMIT :page_size
+   OFFSET :page_offset
```

Comment:
would you mind adding another line break at the end of the file?

Reply:
done